BundleTransport/app/src/main/java/com/ddd/wifidirect/WifiDirectManager.java

```diff
@@ -136,40 +164,42 @@ public void onFailure(int reasonCode) {
      *
      * @param deviceList
      */
-    @Override
+    /*@Override
```

Comment:
i'm wondering about this too. it seems like this has better logic than the requestPeers above.